gulp-core-build-typescript/src/TSLintTask.ts

```diff
@@ -189,4 +195,17 @@ export class TSLintTask extends GulpTask<ITSLintTaskConfig> {
       (this.taskConfig.useDefaultConfigAsBase ? this._defaultLintRules : {}),
       this.taskConfig.lintConfig || {});
   }
+
+  private _writeLintRules(lintRules: any): fs.WriteStream { // tslint:disable-line:no-any
+    const tslintFilePath: string = path.resolve(this.buildConfig.rootPath, this.buildConfig.tempFolder, 'tslint.json');
+    const tslintFileStream: fs.WriteStream = fs.createWriteStream(tslintFilePath);
+
+    const readable: Readable = new Readable();
+    const pipe: fs.WriteStream = readable.pipe(tslintFileStream);
+
+    readable.push(JSON.stringify(lintRules, undefined, 2));
+    readable.push(null); // tslint:disable-line:no-null-keyword
+
+    return pipe;
```

Comment:
Also, it looks like this is the wrong type of stream. Gulp/Vinyl is using a Stream of GulpUtil.Files objects. A better way to accomplish what this function is doing would be to adopt [this](https://onedrive.visualstudio.com/SPPPlat/_git/sp-client?path=%2Fspfx-tools%2Fsp-build-core-tasks%2Fsrc%2Futilities%2FFileUtils.ts&version=GBmaster&_a=contents&line=4&lineStyle=plain&lineEnd=27&lineStartColumn=1&lineEndColumn=2). IMO, we should migrate some of those utilities into directly into GCB.